app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -31,6 +31,12 @@
     private List<Session> mSessions;
     private RecyclerView sessionRecyclerView;
     private String speaker;
+    private static String searchText;
+    private SearchView searchView;
+    final private String SEARCH="searchText";
+    public SpeakersActivity(){
+        searchText=null;
```

Comment:
don't think usually we override constructor in Activities. Not a good practice because activities are created by the system and we cannot be sure of what constructor they are using, plus this method doesn't call `super()`. Anyways thats a secondary thing but usually we don't have a overriding constructor for activity.